project/GatherLicenses.scala

```diff
@@ -1,6 +1,6 @@
-import sbt.Keys._
-import sbt._
-import complete.DefaultParsers._
+import sbt.Keys.*
+import sbt.*
```

Comment:
huh? That's not Scala. I'm a bit surprised this even passes the compiler

Reply:
Hah, interesting, some automated tool did this for me - I'm not sure if this was `scalafmtSbt` or IntelliJ's formatter (likely the latter).

Sure, I will revert this change in my follow up PR.

> Just for fun, note that this **_is_** actually valid Scala. This is the [_preferred_ Scala 3 syntax](https://docs.scala-lang.org/scala3/reference/changed-features/imports.html) - the old `_` syntax is still supported in Scala 3 for compatibility purposes. What is surprising me is that our SBT is happily accepting Scala 3 syntax - I thought it still runs on Scala 2.

Reply:
Note that we already are using the `*` imports in `build.sbt` before that (not my changes):
![image](https://github.com/enso-org/enso/assets/1436948/1d27df3b-e409-4595-bc31-bc40bc98a7db)

Are you sure this needs changing? I guess we should then make this consistent.